js/src/pages/lesson/Header.js

```diff
@@ -4,7 +4,6 @@ import Grid from '@material-ui/core/Grid';
 import Container from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
 import makeStyles from '@material-ui/core/styles/makeStyles';
-import BackLink from '@anu/components/BackLink';
```

Comment:
You can probably delete components/BackLink file if it's not used anywhere else

Reply:
remove component